docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1742,15 +1708,15 @@ This feature is useful for imports which follow a prefix pattern:
 
 
 ### 4.10.3 Integration Testing
-To integration test, there are tests in `ExportOperationTest.java` in `fhir-server-test` module with server integration test cases for system, patient and group export. Further, there are tests in `ImportOperationTest.java` in `fhir-server-test` module. These tests rely on the `fhir-server-config-db2.json` which specifies two storageProviders.
+To integration test, there are tests in `ExportOperationTest.java` in `fhir-server-test` module with server integration test cases for system, patient and group export. Further, there are tests in `ImportOperationTest.java` in `fhir-server-test` module. These tests rely on the `fhir-server-config-postgres.json` which specifies two storageProviders.
```

Comment:
Need to update this because it no longer reflects the fhir-server-test behavior.

Reply:
```suggestion
There are tests in the `fhir-server-test` module for system, patient and group export. These tests rely on fhir-server-config files that currently exist under `fhir-server-webapp/src/test/liberty`.
```

Reply:
new text looks good